entry-generation/godot-entry-generator/src/main/kotlin/godot/entrygenerator/generator/provider/CoreTypeRegistrationValuesHandler.kt

```diff
@@ -0,0 +1,47 @@
+package godot.entrygenerator.generator.provider
+
+import com.squareup.kotlinpoet.ClassName
+import godot.entrygenerator.exceptions.WrongAnnotationUsageException
+import org.jetbrains.kotlin.descriptors.PropertyDescriptor
+import org.jetbrains.kotlin.resolve.BindingContext
+
+class CoreTypeRegistrationValuesHandler(
+    propertyDescriptor: PropertyDescriptor,
+    bindingContext: BindingContext
+) : RegistrationValuesHandler(propertyDescriptor, bindingContext) {
+
+    override fun getPropertyTypeHint(): ClassName {
+
+        return when (propertyHintAnnotation?.fqName?.asString()) {
+            "godot.annotation.ColorNoAlpha" ->
```

Comment:
unrelated but I think the annotation should be `@Color(noAlpha = true)`.

Reply:
I dont think so. Only this typehint exists. In the other case you just export a color with no typehint. No need for a typehint annotation. Same as in gdScript

Reply:
Ahh, that makes sense!